.travis.yml

```diff
@@ -11,7 +11,8 @@ env:
     - SCAN_WORKSPACE="SwiftSampleProject/SwiftSampleProject.xcworkspace"                 SCAN_SCHEME="SwiftyVokoder"              SCAN_SDK="iphonesimulator"
     - SCAN_WORKSPACE="Paging Data Source Example/Paging Data Source Example.xcworkspace" SCAN_SCHEME="Paging Data Source Example" SCAN_SDK="iphonesimulator"
 script: bundle exec scan
-after_script:
+after_success:
+  - bundle exec pod repo update # Travis isn't finding the latest VOKUtilities release: must have an old repo cached
```

Comment:
From the error message that prompted this change:

> Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default.) during validation.

So: probably, but it was part of `pod install` in the past, so it can't be THAT bad.